apps/freelance/src/redux/services/types/user.types.ts

```diff
@@ -34,3 +35,11 @@ export interface AddEducation {
   education_from: string;
   education_to: string;
 }
+
+export interface IUserInfo {
+  email: string;
+  first_name: string;
+  last_name: string;
+  profile_image: null;
+  role: 'Freelancer' | 'JobOwner';
```

Comment:
'Freelancer' | 'JobOwner' - make a separate type for it, you will reuse it often I think

Reply:
Yes, thank you, made a separate type